deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -102,17 +102,14 @@ spec:
     owned:
     - description: MetaOperatorCatalog is the Schema for the metaoperatorcatalogs
         API
-      displayName: MetaOperatorCatalog
       kind: MetaOperatorCatalog
```

Comment:
The `displayName` should be not removed, otherwise the operator will be failed to install.

Is there a way to not update this filed when using `operator-sdk generate csv`?

Reply:
I check the document from operator-sdk,
https://github.com/operator-framework/operator-sdk/blob/master/doc/user/olm-catalog/generating-a-csv.md#updating-your-csv
 displayName fields are overwritten every time when using `operator-sdk generate csv` and they must be added manually.

I think we should remove `operator-sdk generate csv` from makefile `code-gen` and only run it when there are crd changes or upgrade csv version.